lib/cli.js

```diff
@@ -33,6 +33,9 @@ var processArgs = function(argv, options, fs, path) {
 
   if (helper.isString(options.logLevel)) {
     options.logLevel = constant['LOG_' + options.logLevel.toUpperCase()] || constant.LOG_DISABLE;
+  } else if (helper.isDefined(options.logLevel)) {
```

Comment:
@vojtajina It looks like the current behavior, on line 35, is to default unknown options to `LOG_DISABLE` so that if users currently specify `NO` or `OFF` or `FALSE` or `NONE`, etc. then it will just disable. You think we should instead fail if it is unknown?